tools/ci_build/github/azure-pipelines/templates/jar-maven-signing-linux.yml

```diff
@@ -0,0 +1,64 @@
+parameters:
+  - name: JarFileDirectory
+    type: string
+
+  - name: WorkingDirectory
```

Comment:
It'd better to rename this parameter.
It may be confusing

workingDirectory - Working Directory
string.

Specifies the working directory to run commands. If you leave it empty, the working directory is [$(Build.SourcesDirectory)](https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables).

Reply:
good point.

Reply:
1. removed the parameter 'WorkingDirectory'
2. used $(Build.SourcesDirectory) as WorkingDirectory for inline script.